### PR TITLE
Yourls adder

### PR DESCRIPTION
**Pull request by AnmolVirdi**
- Added YOURLS/OFFICIAL
- Updated readme
- Updated in workflow
- Added docker-compose support for selenium tests.

### CODE REVIEW

File: community_images/yourls/official/image.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: yourls
+official_name: YOURLS
+official_website: https://yourls.org/
+source_image_provider: YOURLS
+source_image_repo: docker.io/_/yourls
```
Comment (codervinod) [inline community_images/yourls/official/image.yml:5]: I think _ should be library here. Please check

File: community_images/yourls/official/image.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+source_image_repo_link: https://hub.docker.com/_/yourls
+source_image_readme: https://github.com/docker-library/docs/blob/master/yourls/README.md
+rf_docker_link: rapidfort/yourls
+image_workflow_name: yourls_official
+github_location: yourls/official
+report_url: https://frontrow.rapidfort.com/app/community/imageinfo/docker.io%2Fbitnami%2Fyourls
```
Comment (codervinod) [inline community_images/yourls/official/image.yml:11]: Please fix the link. bitnami should be library

File: community_images/yourls/official/overrides.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+image:
```
Comment (codervinod) [inline community_images/yourls/official/overrides.yml:1]: if you dont have k8s runtime, why do you need overrides file

File: community_images/yourls/official/image.yml (hunk 1 above)
Comment (AnmolVirdi) [reply community_images/yourls/official/image.yml:5]: Rectifying it.

File: community_images/yourls/official/image.yml (hunk 2 above)
Comment (AnmolVirdi) [reply community_images/yourls/official/image.yml:11]: Rectifying it.

File: community_images/yourls/official/overrides.yml (hunk 3 above)
Comment (AnmolVirdi) [reply community_images/yourls/official/overrides.yml:1]: Removing it right now.

**Pull request merged**